test/SILOptimizer/capture_promotion.sil

```diff
@@ -305,4 +305,24 @@ bb0(%0: $*Int, %1 : $<τ_0_0> { var τ_0_0 } <Foo>, %2 : $<τ_0_0> { var τ_0_0
   return %16 : $()
 }
 
+// CHECK-LABL: sil @test_with_dynamic_self
```

Comment:
This has a typo:
```
CHECK-LABL
```

Reply:
@gottesmm  Thanks for catching this!

Reply:
@eeckstein No problem. Happy to help = ).